### PR TITLE
Hive capitalisation fix 

### PR DESCRIPTION
**Pull request by connorjg1**
Fixes the inconsistent capitalisation of "Vaurca Hive" wherever it shows up.

### CODE REVIEW

**Comment by connorjg1** [comment]
!review

**Comment by connorjg1** [comment]
!bugfix

File: html/changelog.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	<tr>
 		<td valign='top'>
 			<div align='center'><font size='3'><b>Space Station 13</b></font></div>
-
```
Comment (SleepyGemmy) [inline html/changelog.html:6]: do not touch the changelog.

**Review by SleepyGemmy — DISMISSED** [review]

File: html/changelog.html (hunk 1 above)
Comment (connorjg1) [reply html/changelog.html]: Addressed

**Pull request merged**